int64 grassNum
int64 xPos
int64 yPos
int64 grassHeight
